android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -691,6 +701,18 @@ public void sendString(String message) {
         }
     }
 
+    public void sendEncodedString(String message) {
```

Comment:
To continue my example, this should be something like:

```java
public void sendBinary(String encodedMessage) {
    if (localDataTrack != null) {
        byte[] binaryMessage = Base64.decode(encodedMessage, Base64.DEFAULT);
        localDataTrack.send(ByteBuffer.wrap(binaryMessage));
    }
}
```

Reply:
Took out sendEncodedString since I realized the ticket only specifies receiving data, but I will keep this in mind when we tackle it.